napari/_qt/widgets/_tests/test_qt_dims.py

```diff
@@ -279,15 +279,20 @@ def test_play_button(qtbot):
     ndim = 3
     view = QtDims(Dims(ndim=ndim))
     qtbot.addWidget(view)
-    button = view.slider_widgets[0].play_button
-    qtbot.mouseClick(button, Qt.LeftButton)
-    qtbot.waitSignal(view._animation_thread.started, timeout=5000)
-
-    with qtbot.waitSignal(view._animation_thread.finished, timeout=7000):
-        qtbot.mouseClick(button, Qt.LeftButton)
+    slider = view.slider_widgets[0]
+    button = slider.play_button
 
+    # Need looping playback so that it does not stop before we can assert that.
+    assert slider.loop_mode == 'loop'
     assert not view.is_playing
 
+    qtbot.mouseClick(button, Qt.LeftButton)
+    qtbot.waitUntil(lambda: view.is_playing)
```

Comment:
```suggestion
    qtbot.waitUntil(view.is_playing)
```

Reply:
I originally added this but then reverted because `QtDims.is_playing` is a property, not a function/callable.